region/region-readme.md

```diff
@@ -82,6 +82,16 @@ The requirements to destroy an obstacle are found in two places. If a situation
 __Additional considerations:__ 
 Obstacles are not systematically represented in the model. They are put in as needed, in rooms where there could be a reason to pass by an obstacle twice without exiting.
 
+### Enemies
+A room can have an array of enemies. This is the list of enemies that may be present in the room. They may be relevant for farming purposes. Each `enemy` object can have the following properties:
```

Comment:
I would like something along the lines of "They may be relevant for farming purposes or for estimating room difficulty."

Reply:
Done